weather-a127-resources/api/swagger/swagger.yaml

```diff
@@ -82,8 +82,7 @@ paths:
         default:
           description: Error
           schema:
-<<<<<<< HEAD
```

Comment:
That HEAD line was there and I removed it.  Is there any problem merging
the file without that line.  It's causing a syntax error.

On Sat, Jan 31, 2015 at 3:46 PM, Scott Ganyo <notifications@github.com>
wrote:

> In weather-a127-resources/api/swagger/swagger.yaml
> <https://github.com/apigee-127/a127-samples/pull/11#discussion_r23893361>:
>
> > @@ -82,8 +82,7 @@ paths:
> >          default:
> >            description: Error
> >            schema:
> > -<<<<<<< HEAD
>
> It looks like you have a conflict to resolve.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apigee-127/a127-samples/pull/11/files#r23893361>.
>



-- 
*Mike Malloy* | *apigee <https://apigee.com/>* | m: +1.650.815.1769 |
twitter @ <http://twitter.com/mmalloyapigee>mmalloyapigee @apigee
<https://twitter.com/apigee>